contracts/modules/STO/POLYCappedSTO.sol

```diff
@@ -0,0 +1,526 @@
+    event SetAccredited(address _investor, bool _accredited);
+   /**
+    * Event for token purchase logging
+    * @param _purchaser who paid for the tokens
+    * @param _beneficiary who got the tokens
+    * @param _value amount paid for purchase in Fund Raise Type
+    * @param _valueUSD value of POLY invested in USD at the time of purchase
+    * @param _amount amount of tokens purchased
+    */
+    event TokenPurchase(
+        address indexed _purchaser,
+        address indexed _beneficiary,
+        uint256 _value,
+        uint256 _valueUSD,
+        uint256 _amount
+    );
+    event FundsReceived(
+        address indexed _purchaser,
+        address indexed _beneficiary,
+        FundRaiseType _fundRaiseType,
+        uint256 _receivedValue,
+        uint256 _spentValue
+    );
+    event ReserveTokenMint(address indexed _owner, address indexed _treasuryWallet, uint256 _tokens);
+    event SetWallet(address _oldWallet, address _newWallet);
+    event SetTreasuryWallet(address _oldWallet, address _newWallet);
+    event SetLimits(uint256 _minimumInvestment, uint256 _nonAccreditedLimit, uint256 _maxNonAccreditedInvestors);
+    event SetTimes(uint256 _startTime, uint256 _endTime);
+    event SetRate(uint256 _rate);
+    event SetCap(uint256 _cap);
+
+    ///////////////
+    // Modifiers //
+    ///////////////
+
+    modifier notStarted() {
+        /*solium-disable-next-line security/no-block-members*/
+        require(now < startTime, "STO already started");
+        _;
+    }
+
+    ///////////////////////
+    // STO Configuration //
+    ///////////////////////
+
+    constructor (address _securityToken, address _polyAddress) public
+    Module(_securityToken, _polyAddress)
+    {
+    }
+
+    /*
+     * @notice Function used to intialize the contract variables
+     * @param _startTime Unix timestamp at which offering get started
+     * @param _endTime Unix timestamp at which offering get ended
+     * @param _cap Maximum No. of token base units for sale
+     * @param _rate Token units a buyer gets multiplied by 10^18 per wei / base unit of POLY
+     * @param _minimumInvestment Minimun investment in fund raise type (* 10**18)
+     * @param _nonAccreditedLimit Limit in fund raise type (* 10**18) for non-accredited investors
+     * @param _maxNonAccreditedInvestors Maximum number of non-accredited investors allowed (0 = unlimited)
+     * @param _wallet Ethereum account address to hold the funds
+     */
+    function configure(
+        uint256 _startTime,
+        uint256 _endTime,
+        uint256 _cap,
+        uint256 _rate,
+        uint256 _minimumInvestment,
+        uint256 _nonAccreditedLimit,
+        uint256 _maxNonAccreditedInvestors,
+        address payable _wallet
+    )
+        external
+        onlyFactory
+    {
+        require(endTime == 0, "Already configured");
+        FundRaiseType[] memory _fundRaiseTypes = new FundRaiseType[](1);
+        _fundRaiseTypes[0] = FundRaiseType.POLY;
```

Comment:
As this module only raises in Poly, I think it might be best to just drop the whole FundRaiseType for clarity.

Reply:
Ok I can do that. `fundRasiedTypes` will still be inherited from STO.sol so I thought it might have been better to set it as POLY instead of not setting it in case it caused confusion or the dapp required this info.